### DISCUSSION TITLE
Suggestions

### DISCUSSION THREAD

**Discussion by dmtamayo92** [NickvisionApps/Parabolic#799] [Ideas]
I think it should be possible to choose the temporary folder where downloads are made.
In the separate chapters option, you can choose to automatically rename the name to the name of each chapter.

**Comment by nlogozzo** [comment]
> I think it should be possible to choose the temporary folder where downloads are made.

What is the need of a temporary download folder if it will be moved to the save folder anyway?

> In the separate chapters option, you can choose to automatically rename the name to the name of each chapter.

Please create a feature request issue for this.

**Reply by Quizzard2** [reply]
I think there should be an option to change the download folder

**Reply by nlogozzo** [reply]
> I think there should be an option to change the download folder

As in where the part and temp files are downloaded? Or where the final download saves to? Because the later is already an option...

As for the former, why?

**Comment by Threat-Conductor** [comment]
please add a seperate tab to distiguish between completed and failed downloads

**Reply by Threat-Conductor** [reply]
also add the option to ignore anything marked [private video] or [deleted video] and the ability to cancel downloads from the failed downloads tab

**Reply by nlogozzo** [reply]
> the ability to cancel downloads from the failed downloads tab

How would you cancel downloads that have failed? They are completed once they failed nothing to cancel...

**Comment by laobanpi** [comment]
抖音视频是什么链接呢，我复制的链接显示不对呢
